brand/models/commentary.py

```diff
@@ -182,6 +219,27 @@ def compute_inherited_rating(self, inheritance_set=None, throw_error=False):
 
     feature_refresh_date = models.DateTimeField(null=True, blank=True)
     feature_json = models.JSONField(null=True, blank=True, default=dict)
+    feature_override = YAMLField(
```

Comment:
This field should be hidden on the admin panel for now or the ratings team is going to think it's a bug.